src/uct/sm/mm/base/mm_iface.c

```diff
@@ -772,3 +785,28 @@ UCS_CLASS_DEFINE(uct_mm_iface_t, uct_base_iface_t);
 UCS_CLASS_DEFINE_NEW_FUNC(uct_mm_iface_t, uct_iface_t, uct_md_h, uct_worker_h,
                           const uct_iface_params_t*, const uct_iface_config_t*);
 static UCS_CLASS_DEFINE_DELETE_FUNC(uct_mm_iface_t, uct_iface_t);
+
+static UCS_CLASS_INIT_FUNC(uct_mm_ext_iface_t, uct_md_h md, uct_worker_h worker,
+                           const uct_iface_params_t *params,
+                           const uct_iface_config_t *tl_config)
+{
+    uct_mm_ext_iface_config_t *config =
+                    ucs_derived_of(tl_config, uct_mm_ext_iface_config_t);
+
+    UCS_CLASS_CALL_SUPER_INIT(uct_mm_iface_t, md, worker, params, tl_config);
+
+    self->super.config.errh_caps = (config->error_handling == UCS_CONFIG_ON) ?
```

Comment:
maybe just `if`, because `errh_caps` was already set to `0ul` in the `uct_mm_iface_t` ctor

Reply:
i'd leave it here to properly handle config value in one place and do not rely on another part of code, which can be changed in the future